### ISSUE TITLE
driver.findElement is unable to find some elements in ver 1.3.0

### ISSUE THREAD

**Issue by robert-altom** [robert-altom/test#744]
**In what language are the test written?** 

Java

**What version of AltUnityTester are you using?**

1.3.0

**On what device are you trying to run the test?**

Android

**The OS from where you are running the test?**

Windows

### Summary

I recently updated to ver 1.3.0 of the Unity asset as well as the Java bindings. I've noticed that driver.findElement and driver.waitForElement do not work with some of the elements any more. It returns Exception thrown: error:notFound.
But If I use driver.findElementWhereNameContains, then the driver finds the same elements.

Furthermore, if I use ver 1.3.0 Java bindings with an apk made with ver 1.2.2 Unity assets, then the methods work fine too.

---
<sub>You can find the original issue from GitLab [here](https://gitlab.com/altom/altunity/altunitytester/-/issues/150).</sub>

**Comment by robert-altom** [comment]
Hi Robert, I have tried a few more tests and I have more info.
The object does not have a strange name. It's called "SocialButton" and it's the only object of that name in any of the loaded scenes. 
And yes, there are multiple scenes loaded.

I have noticed that `driver.findElement` in v1.3.0 can find the objects in the current scene. But if I try to find an object in another loaded scene, it throws `ro.altom.altunitytester.altUnityTesterExceptions.NotFoundException`.
But `driver.findElementWhereNameContains` in v1.3.0 is able to find those same objects in scenes that are not the current scene.

In v1.2.2, `driver.findElement` is able to find objects in any loaded scene regardless of it being the current scene or not.
